test/DualTest.jl

```diff
@@ -589,7 +585,7 @@ ForwardDiff.:≺(::Type{OuterTestTag}, ::Type{TestTag}) = false
         @test pq isa Tuple{Dual{TestTag()},Dual{TestTag()}}
         # We have to adjust tolerances if lower accuracy is requested
         # Therefore we don't use `dual_isapprox`
-        tol = V === Float32 ? 5f-4 : 1e-6
+        tol = V === Float32 ? 5f-4 : 1e-5
```

Comment:
This seems unrelated?

Reply:
I think this is an intermittent failure on 1.8, e.g. here:

https://github.com/JuliaDiff/ForwardDiff.jl/actions/runs/3601997063/jobs/6068433780#step:6:445